integration-tests/container/benchmark/Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM phoronix/pts
+FROM --platform=linux/amd64 phoronix/pts as stage-amd64
+FROM --platform=linux/ppc64le quay.io/rhacs-eng/phoronix-pts-ppc64le as stage-ppc64le
 
+# Declare TARGETARCH to make it available
+ARG TARGETARCH
+
+# Select final stage based on TARGETARCH ARG
+FROM stage-${TARGETARCH} as final
```

Comment:
If this is just for CI then this should work there, but tricks such as this will not work for downstream builds.

Reply:
Not sure why this wont work?